velox/exec/TableScan.cpp

```diff
@@ -291,7 +299,7 @@ void TableScan::checkPreload() {
           [executor, this](std::shared_ptr<connector::ConnectorSplit> split) {
             preload(split);
 
-            executor->add([taskHolder = operatorCtx_->task(), split]() mutable {
+            executor->add([split]() mutable {
```

Comment:
The task holder was added in https://github.com/facebookincubator/velox/pull/4849. So we don't need it anyway?

Reply:
Yes, I think so, please see https://github.com/facebookincubator/velox/issues/8198#issuecomment-1876999843